### DISCUSSION TITLE
プライベートコンテナで拡張機能を無効にする機能

### DISCUSSION THREAD

**Discussion by newmomizi** [Floorp-Projects/Floorp#477] [Feature Request]
Floorp 11.4.0から「プライベートコンテナ」機能が追加されましたが、このプライベートコンテナではプライベートウィンドウと異なり、Floorpにインストールしている全ての拡張機能が動作してしまいます。
そのため、プライバシーやセキュリティの観点で問題があります。

「プライベートウィンドウでの実行」チェックを入れていない拡張機能に限り、プライベートコンテナで開いているページにアクセスできなくすることなどは可能でしょうか。
よろしくお願いします。

**Comment by Ryosuke-Asano** [comment]
ん－難しいねぇ。

調べてみるわ
